cli/internal/clusterid/id.go

```diff
@@ -63,3 +64,8 @@ func (f *File) Merge(other File) *File {
 
 	return f
 }
+
+// GetClusterName returns the name of the cluster.
+func GetClusterName(cfg *config.Config, idFile File) string {
```

Comment:
I can see your point about package boundary from a purist point of view, but I honestly think that it's okay to entangle them since they are `internal` + clusterid is a tiny pkg with with one struct. But of course I can make it an interface :) I'm just wondering what we gain. Are we ever wanting to inject a another config? `clusterid` is also a very tiny dependency to import